consensus/src/runway/mod.rs

```diff
@@ -743,7 +690,16 @@ where
 
                 notification = self.notifications_from_alerter.next() => match notification {
                     Some(notification) => {
-                        self.on_alert_notification(notification)
+                        trace!(target: "AlephBFT-runway", "Received alerter notification: {:?}.", notification);
```

Comment:
To keep it consistent with the other `select` branches, maybe let's include this logic in some `fn handle_alerter_notification`

Reply:
If you don't want to spam with too many functions, the logic of `handle_validation_error` can be fully included in `handle_alerter_notification`

Reply:
:D You are so very right that I just 20mins ago did that within the next PR. Can we leave it here as-is for now?

Reply:
sure